CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ Versions
 * Use only major Node version to name node images
 * Add Node 12
 * Upgrade ScoutSuite to 5.3.3
+* Upgrade Java: 8u222, 11.0.4
```

Comment:
Can you please amend the existing line 11 instead?

Reply:
Done :)